README.md

```diff
@@ -18,6 +18,35 @@ To get updates, watch the repo and follow the [Twitter account](https://twitter
 
 # Tips list
 
+## #12 - Tip to measure performance of a javascript block
+
+2016-01-12 by [@manmadareddy](https://twitter.com/manmadareddy)
+
+For quickly measuring performance of a javascript block, we can use the console functions like
+console.time(label) and console.timeEnd(label).
```

Comment:
add link [`console.timeEnd(label)`](https://developer.mozilla.org/en-US/docs/Web/API/Console/timeEnd) and format